README.md

```diff
@@ -1 +1,177 @@
+
+The `.browserlist` config is provided.
+
+## Key concepts
+
+Undercoat is based on the next main principles:
+
+- [BEM methodology](#BEM-methodology)
+- [No default theme](#No-default-theme)
+- [Mobile friendly](#Mobile-friendly)
+- [SEO friendly](#SEO-friendly)
+- [WCAG 2.0 compliance](#WCAG-20-compliance)
+
+### BEM methodology
+
+Undercoat provides a set of common page components (breadcrumbs, cards, form elements etc.). Each component is logically and functionally independent [block](https://en.bem.info/methodology/key-concepts/#block). Blocks allow for their re-use, as well as facilitating the project development and support process. Read more about blocks in official BEM [documentation](https://en.bem.info/methodology/key-concepts/#block-features).
+
+Undercoat follows BEM naming convention with the [classic naming scheme](https://en.bem.info/methodology/naming-convention/#naming-rules), where the name of an entity is unique. The same BEM entity always has the same name in all technologies (CSS, JavaScript, HTML etc.). 
+
+Undercoat follows an alternative file structure approach to organizing the framework's file system. It's [nested](https://en.bem.info/methodology/filestructure/#nested) on the root level and [flat](https://en.bem.info/methodology/filestructure/#flat) on the component level.
+
+**Example**
+
+```sh
+project
+    common.blocks/
+        input/                                # Directory for the input block 
+            input_type_search.css             # CSS implementation of the input_type modifier 
+            input__clear_visible.css          # CSS implementation of the input__clear_visible modifier 
+            input__clear.css                  # CSS implementation of the input__clear element 
+            input__clear.js                   # JavaScript implementation of the input__clear element 
+            input.css                         # CSS implementation of the input block 
+            input.js                          # JavaScript implementation of the input block 
+```
+
+### No default theme
+
+Undercoat doesn't provide a default theme for the components. Their CSS code represents only their layout composition, not the appearance. So there's no need for a developer to override the framework styles. Just write (extend) your own code (add new properties) next to the component code following BEM's redefinition levels [rules](https://en.bem.info/methodology/redefinition-levels).
+
+**Example**
+
+```sh
+project/
+    common.blocks/       # redefinition level with framework's blocks 
+        breadcrumbs/     # breadcrumbs block composition styles
+    project.blocks/      # redefinition level with project blocks 
+        breadcrumbs/     # breadcrumbs block project appereance styles
+```
+
+Just remember Undercoat doesn't impose design decisions that you have to fight to undo!
+
+### Mobile friendly
+
+Following actual global trends, Undercoat was built based on mobile first-approach. 
```

Comment:
"Undercoat __is based__ on the _mobile-first approach_." --> we haven't built a framework yet :) Also as I think if we finish this we need to always maintenance code so there is can't be the Past time

Reply:
You're right!